### PR TITLE
Renamed artifact group id to com.linkedin.hive and packaging changed to gobblin-hive from apache-hive.

### PR DESCRIPTION
**Pull request by abhisheknath2011**
### What changes were proposed in this pull request?
1. This is artifact group id and packing change PR for the AvroCompatibilityHelper support PR - https://github.com/linkedin/gobblin-hive/pull/1
2. Renamed artifact group id to com.linkedin.hive from org.apache.hive and packaging changed to gobblin-hive from apache-hive.

### Why are the changes needed?
1. These changes were need to make a release for the changes in PR - https://github.com/linkedin/gobblin-hive/pull/1.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Local build

### CODE REVIEW

File: packaging/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.hive</groupId>
+    <groupId>com.linkedin.hive</groupId>
     <artifactId>hive</artifactId>
     <version>1.0.1</version>
```
Comment (srramach) [inline packaging/pom.xml:22]: Don't we want to change this to `1.0.1-avro`? Or, do you intend to do it in a separate PR?

Comment (abhisheknath2011) [reply packaging/pom.xml:22]: Planning to do in a separate PR. But can also add as a new commit in the same PR. Please let me know.

Comment (srramach) [reply packaging/pom.xml:22]: Separate PR is fine.

**Review by srramach — APPROVED** [review]
